contracts/FungibleToken.cdc

```diff
@@ -109,6 +109,30 @@ pub contract interface FungibleToken {
         /// @param from: The Vault resource containing the funds that will be deposited
         ///
         pub fun deposit(from: @Vault)
+
+        /// Below is referenced from the FLIP #69 https://github.com/onflow/flips/blob/main/flips/20230206-fungible-token-vault-type-discovery.md
+        /// 
+        /// Returns the Vault types that the the receiver is able to accept in its `deposit` method
+        /// this then it would return `[Type<@FlowToken.Vault>()]` and if any custom receiver
+        /// uses the default implementation then it would return empty array as its parent
+        /// resource doesn't conform with the `FungibleToken.Vault` resource.
+        ///
+        /// Custom receiver implementations are expected to upgrade their contracts to add an implementation
+        /// that supports this method because it is very valuable for various applications to have.
+        ///
+        /// @return list of supported deposit vault types by the implementing resource.
+        /// 
+        pub fun getSupportedVaultTypes(): [Type] {
```

Comment:
Thinking about this more, I would like the return type of this to be `{Type: Bool}`. You can still iterate through it with `return.keys`, and it makes checking a specific type much easier. Do you think you'll be able to make that change in time for the upgrade next week?